### ISSUE TITLE
Error in local dev: "This likely means that the library which declares NgxDaterangepickerMd has not been processed correctly by ngcc..."

### ISSUE THREAD

**Issue by manwithtwowatches** [fetrarij/ngx-daterangepicker-material#402]
**Versions**
* Angular version: 11
* component version: 5.0.1

**Describe the bug**
I am trying to use a local version of the package. Having npm installed and built ngx-daterangepicker-material, and then included in a fresh angular project package.json with

`"ngx-daterangepicker-material": "file:../ngx-daterangepicker-material"`

and included it in app.module with

`import { NgxDaterangepickerMd } from '../../node_modules/ngx-daterangepicker-material'
imports: [
    NgxDaterangepickerMd.forRoot(),`

when I try to serve the app I get the error:

`Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

This likely means that the library which declares NgxDaterangepickerMd has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.`

at line 3 of daterangepicker.module.ts at
`export declare class NgxDaterangepickerMd { `

Having read around a bit it seems these kind of errors usually happen when you're using the wrong version of Angular, however it seems to me that everything is using version 11.2.1x

Basically I'm trying to set up a dev environment. Does anyone have any idea why this might be happening?

**Comment by umangchadha** [comment]
Hi facing same issue when i am trying to run my code on stackblitz. any luck?
